src/Tests/Toolchain/ToolchainEnv.test.ts

```diff
@@ -14,14 +14,29 @@
  * limitations under the License.
  */
 
-import {assert} from 'chai';
+import {assert, expect} from 'chai';
+import {ToolchainInfo} from '../../Backend/Toolchain';
+import {DebianToolchain} from '../../Backend/ToolchainImpl/DebianToolchain';
+import {Job} from '../../Job/Job';
 
-import {ToolchainEnv} from '../../Toolchain/ToolchainEnv';
+import {Env, ToolchainEnv} from '../../Toolchain/ToolchainEnv';
 import {MockCompiler} from '../MockCompiler';
+import {MockFailedJob, MockJob} from '../MockJob';
 
 suite('Toolchain', function() {
+  suite('Env', function() {
+    suite('#build()', function() {
+      test('check prepared', function() {
```

Comment:
Maybe we can change it as NEG? :D

```suggestion
      test('NEG: check prepared with clear jobs', function() {
```